### PR TITLE
fix issue where configdumpimporter parses wrong type

### PR DESCRIPTION
**Pull request by cgardens**
pretty self explanatory, see issue below.

### CODE REVIEW

File: airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java [hunk 1]
```diff
@@ -484,25 +485,25 @@ public void importIntoWorkspace(String targetVersion, UUID workspaceId, File arc
...
+            } catch (final ConfigNotFoundException e) {
               return;
             }
             configRepository.writeStandardSync(standardSync);
           });
     }
   }
 
-  protected <T> void importSourceDefinitionIntoWorkspace(Stream<T> configs)
+  protected <T> void importSourceDefinitionIntoWorkspace(final Stream<T> configs)
```
Comment (cgardens) [inline airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java:501]: @ChristopheDuong i think this just a typo where this class is supposed to be pulling standard source definitions but instead pulls standard sources.

Comment (ChristopheDuong) [reply airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java:501]: the method is named (confusingly) `listStandardSources` but returned a `list<StandardSourceDefinitions>` though:

https://github.com/airbytehq/airbyte/blob/da3677e821afc8a385c4c937692a9b039425b8fb/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java#L152

Comment (ChristopheDuong) [reply airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java:501]: Shouldn't it be removed completely now that you added `listStandardSourceDefinitions`?

Comment (cgardens) [reply airbyte-server/src/main/java/io/airbyte/server/ConfigDumpImporter.java:501]: whacky!

**Pull request merged**